docs/current_docs/api/arguments.mdx

```diff
@@ -133,6 +133,84 @@ The result will look like this:
 HELLO, WORLD
 ```
 
+## Integer arguments
+
+To pass an integer argument to a Dagger function, add the corresponding flag to the `dagger call` command, followed by the integer value.
+
+Here is an example of a Dagger function that accepts a integer argument:
+
+<Tabs groupId="language">
+
+<TabItem value="Go">
+```go file=./snippets/functions/arguments-integer/go/main.go
+```
+</TabItem>
+
+<TabItem value="Python">
+```python file=./snippets/functions/arguments-integer/python/main.py
+```
+</TabItem>
+
+<TabItem value="TypeScript">
+```typescript file=./snippets/functions/arguments-integer/typescript/index.ts
+```
+</TabItem>
+
+</Tabs>
+
+Here is an example call for this Dagger Function:
+
+```shell
+dagger call add-integer --a 1 --b 2
+```
+
+The result will look like this:
+
+```shell
+3
+```
+
+## Float arguments
+
+:::warning
+Floats are currently only supported in Go and TypeScript SDKs for now.
```

Comment:
The information in this section is only applicable to the Go and TypeScript SDKs. Floating-point number arguments are not currently supported in the Go SDKs.

Reply:
Actually it's now supported in Python so I'll remove this msg :D